app/services/prior_authority/assessment_syncer.rb

```diff
@@ -53,6 +57,14 @@ def sync_additional_cost(cost, cost_data, application)
         )
       end
 
+      def sync_further_info_request(application, app_store_record)
+        data = app_store_record['application']
+        application.update(
+          further_information_explanation: data['further_information_explanation'],
```

Comment:
So there's a slight complication here: If the caseworker checks 'further information', fills in some further information in the box that appears, then changes their mind and _unchecks_ further information, `data['further_information_explanation']` will still be populated, even though no further information is needed. As a safeguard, we should look at `data['updates_needed']` to see what fields are explicitly asked for, rather than just inferring it from what explanation texts have been populated.